### PR TITLE
{bio}[foss/2015a] cutadapt 1.8.1

### PR DESCRIPTION
**Pull request by jordiblasco**

### CODE REVIEW

**Comment by verdurin** [comment]
Worth noting that #2569 has been merged, which is 1.9.1 for foss-2015b.

**Comment by boegel** [comment]
Jenkins: ok to test

File: easybuild/easyconfigs/c/cutadapt/cutadapt-1.8.1-foss-2015a-Python-2.7.9.eb [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+pyshortver = '.'.join(pyver.split('.')[:2])
+
+versionsuffix = "-%s-%s" % (python, pyver)
+
+dependencies = [
+    (python, pyver),
+]
+
+sanity_check_paths = {
+    'files': ['bin/cutadapt','lib/python%s/site-packages/cutadapt/_align.so' % pyshortver],
```
Comment (boegel) [inline easybuild/easyconfigs/c/cutadapt/cutadapt-1.8.1-foss-2015a-Python-2.7.9.eb]: should be using `SHLIB_EXT` instead of hardcoding `.so`

**Comment by boegel** [comment]
superseded by #2569 and more recent easyconfigs for `cutadapt`

**Pull request closed without merging**